recipes/libusb/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "libusb" %}
+{% set version = "1.0.22" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: 'https://github.com/libusb/libusb/releases/download/v{{ version}}/{{ name }}-{{ version }}.tar.bz2'
+  sha256: 75aeb9d59a4fdb800d329a545c2e6799f732362193b465ea198f2aa275518157
+
+build:
+  number: 0
+  run_exports:
+    - {{ pin_subpackage('libusb') }}
+
+host:
```

Comment:
Think this should be under `requirements`.

Reply:
Resolved in 4a14d75